### PR TITLE
Clarify rate limits for special "id" or "login" per minute limits

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Clarified that the exceptions to a general rate limit are for the listed URL only, not extensions of it.

### Resolves:
[OKTA-165417](https://oktainc.atlassian.net/browse/OKTA-165417)

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
@jaikumarbathija-okta The problem is, customers think /api/v1/apps/{id}/users falls under 

Get, update, or delete an application | /api/v1/apps/{id} | 500

because of the {id}. If I wrote it so that /api/v1/apps/{id} exactly is the only one that has 500, would that be correct?

**Comment by jaikumarbathija-okta** [comment]
@mystiberry-okta - That is accurate, the only one that would fall under the /api/v1/apps bucket, but has been kept out as an exception and set to 500 is /api/v1/apps/{id}. Everything else that has a prefix /api/v1/apps falls under the larger bucket, which has a limit of 100.

**Comment by mystiberry-okta** [comment]
@jaikumarbathija-okta  I updated the table to try and clarify. Is this accurate?  

image

File: _source/_docs/api/getting_started/rate-limits.md [hunk 1]
```diff
@@ -29,21 +29,23 @@ When reading the following tables, remember that a more specific limit is consid
...
+| Create or list applications | `/api/v1/apps`   except `/api/v1/apps/{id}`                                     |   100 |
+| Get, update, or delete an application | `/api/v1/apps/{id}` only   |   500 |
+| Authenticate different end users | `/api/v1/authn`                       |   500 |
+| Creating or listing groups | `/api/v1/groups` except  `/api/v1/groups/{id}` |  500 |
+| Get, update, or delete a group | `/api/v1/groups/{id}` only          | 1000 |
+| Get System Log data | `/api/v1/logs`                                           | 120 |
+| Get session information | `/api/v1/sessions`                               |   750 |
+| Create or list users | `/api/v1/users` except `/api/v1/users/{id}` and `/api/v1/users/{login}`    |   600 |
+| Get a user by user ID | `/api/v1/users/{id}`  only                           | 2000 |
+| Get a user by user login name | `/api/v1/users/{login}`  only            | 2000 |
```
Comment (prasanthmathialagan-okta) [inline _source/_docs/api/getting_started/rate-limits.md]: It is misleading to list "Get a user by user ID" and "Get a user by user login name" separately each with a limit of 2000. Actually it is collectively 2000 for "Get a user by user ID or login name".

Comment (mystiberry-okta) [reply _source/_docs/api/getting_started/rate-limits.md]: ack, thanks!

File: _source/_docs/api/getting_started/rate-limits.md [hunk 2]
```diff
@@ -29,21 +29,22 @@ When reading the following tables, remember that a more specific limit is consid
...
+| Create or list applications | `/api/v1/apps`   except `/api/v1/apps/{id}`                                     |   100 |
+| Get, update, or delete an application | `/api/v1/apps/{id}` only   |   500 |
+| Authenticate different end users | `/api/v1/authn`                       |   500 |
+| Creating or listing groups | `/api/v1/groups` except  `/api/v1/groups/{id}` |  500 |
+| Get, update, or delete a group | `/api/v1/groups/{id}` only          | 1000 |
+| Get System Log data | `/api/v1/logs`                                           | 120 |
+| Get session information | `/api/v1/sessions`                               |   750 |
+| Create or list users | `/api/v1/users` except `/api/v1/users/{id}` and `/api/v1/users/{login}`    |   600 |
+| Get a user by user ID or login (combined) | `/api/v1/users/{id}` or `/api/v1/users/{login}`  only   | 2000 |
+| Create, update, or delete a user by ID | `/api/v1/users/{id}` only     |   600 |
```
Comment (prasanthmathialagan-okta) [inline _source/_docs/api/getting_started/rate-limits.md]: One minor nit.

/api/v1/users/{id} is used for updating and deleting a user. For user creation, /api/v1/users is used

Comment (mystiberry-okta) [reply _source/_docs/api/getting_started/rate-limits.md]: will fix that, thanks!

**Pull request merged**